docs/how-to/index.rst

```diff
@@ -9,6 +9,7 @@ Get started with Real-time Ubuntu:
    :maxdepth: 1
 
    Enable Real-time Ubuntu <enable-real-time-ubuntu>
+   Revert from Real-time Ubuntu to Generic Ubuntu <revert-from-rt-kernel>
```

Comment:
```suggestion
   Switch from real-time to generic kernel <revert-from-rt-kernel>
```
The existing "Enable Real-time Ubuntu" is strange. Should this also be Disable Real-time Ubuntu?